cypress/integration/markdown/markdown_text_spec.js

```diff
@@ -67,24 +66,20 @@ describe('Markdown message', () => {
-            {name: 'Markdown - in-line images 3 (Gif)', fileKey: 'markdown_inline_images_3'},
-            {name: 'Markdown - in-line images 4 (4k)', fileKey: 'markdown_inline_images_4'},
-            {name: 'Markdown - in-line images 5 (Panorama)', fileKey: 'markdown_inline_images_5'},
-        ];
+    it('Markdown - block quotes 2', () => {
+        const baseUrl = Cypress.config('baseUrl');
+        const expectedHtml = `<h3 class="markdown__heading">Block Quotes</h3><p><strong>The following markdown should render within the block quote:</strong></p>
+<blockquote>
+<h4 class="markdown__heading">Heading 4</h4><p><em>Italics</em>, <em>Italics</em>, <strong>Bold</strong>, <strong><em>Bold-italics</em></strong>, <strong><em>Bold-italics</em></strong>, <del>Strikethrough</del>
+<span data-emoticon="slightly_smiling_face"><span alt=":slightly_smiling_face:" class="emoticon" title=":slightly_smiling_face:" style="background-image: url(&quot;${baseUrl}/static/emoji/1f642.png&quot;);"></span></span> <span data-emoticon="slightly_smiling_face"><span alt=":slightly_smiling_face:" class="emoticon" title=":slightly_smiling_face:" style="background-image: url(&quot;${baseUrl}/static/emoji/1f642.png&quot;);"></span></span> <span data-emoticon="wink"><span alt=":wink:" class="emoticon" title=":wink:" style="background-image: url(&quot;${baseUrl}/static/emoji/1f609.png&quot;);"></span></span> <span data-emoticon="scream"><span alt=":scream:" class="emoticon" title=":scream:" style="background-image: url(&quot;${baseUrl}/static/emoji/1f631.png&quot;);"></span></span> <span data-emoticon="bamboo"><span alt=":bamboo:" class="emoticon" title=":bamboo:" style="background-image: url(&quot;${baseUrl}/static/emoji/1f38d.png&quot;);"></span></span> <span data-emoticon="gift_heart"><span alt=":gift_heart:" class="emoticon" title=":gift_heart:" style="background-image: url(&quot;${baseUrl}/static/emoji/1f49d.png&quot;);"></span></span> <span data-emoticon="dolls"><span alt=":dolls:" class="emoticon" title=":dolls:" style="background-image: url(&quot;${baseUrl}/static/emoji/1f38e.png&quot;);"></span></span></p>
+</blockquote>`;
 
-        tests.forEach((test) => {
-            it(test.name, () => {
-                // #  Post markdown message
-                cy.postMessageFromFile(`markdown/${test.fileKey}.md`);
+        // #  Post markdown message
+        cy.postMessageFromFile('markdown/markdown_block_quotes_2.md');
 
-                // * Verify that HTML Content is correct.
-                // Note we use the Gigantic timeout to ensure that the large images will load
-                cy.compareLastPostHTMLContentFromFile(`markdown/${test.fileKey}.html`, TIMEOUTS.GIGANTIC);
-            });
+        // * Verify that HTML Content is correct
+        cy.getLastPostId().then((postId) => {
+            cy.get(`#postMessageText_${postId}`, {timeout: TIMEOUTS.MEDIUM}).should('have.html', expectedHtml);
         });
     });
```

Comment:
Markdown tests for text and in-line images were separated.  Intention is to separate the spec group in preparation for parallel testing.